frontend/src/components/Markers.jsx

```diff
@@ -0,0 +1,173 @@
+import "../style.css";
+import React, { useState, useEffect } from "react";
+import { MapContainer, TileLayer, Marker, Popup } from "react-leaflet";
+import { Icon } from "leaflet";
+import "leaflet/dist/leaflet.css";
+
+function Markers() {
+  // créaton de marqueurs stylisés
+  const customIcon = new Icon({
+    iconUrl: "./src/assets/pattecoeur.png",
+    iconSize: [38, 38],
+  });
+  // création d'un tableau pour y stocker les images des popups de la carte
+  const popUp = [
```

Comment:
Please move these datas in a specific folder (ex: data)

Reply:
Fixed it Ayoub, it's gone :)